### PR TITLE
refactor: new member notification to owner

### PR DESCRIPTION
**Pull request by sshanzel**
Update the notification we send to owners when users join the Squad.

### CODE REVIEW

**Review by idoshamun — APPROVED** [review]
Please merge only once we release v5

File: src/notifications/generate.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ export const notificationTitleMap: Record<
   squad_member_joined: (
     ctx: NotificationSourceContext & NotificationDoneByContext,
   ) =>
-    `Your squad is growing! <b>${ctx.doneBy.name}</b> has <span class="text-theme-color-cabbage">joined</span> <b>${ctx.source.name}</b>.`,
+    `Your squad <b>${ctx.source.name}</b> is <span class="text-theme-color-cabbage">growing</span>! Welcome <b>${ctx.doneBy.name}</b> to the squad with an upvote or comment.`,
```
Comment (idoshamun) [inline src/notifications/generate.ts:68]: Are you sure the copy is `with an upvote or comment`? I can't welcome with an upvote. Worth checking with product

Comment (rebelchris) [reply src/notifications/generate.ts:68]: Copy mentions: `Your squad {{{squad_name}}} is growing.  Welcome {{{new username}}} to your squad with a comment.`

**Review by rebelchris — CHANGES_REQUESTED** [review]
Also this action should be different right?
It should prompt to the comment being open immediate which seems to be missing now.

File: src/notifications/generate.ts (hunk 1 above)
Comment (sshanzel) [reply src/notifications/generate.ts:68]: Oh, I referred to the Sketch file. Will verify which is the latest.

Screenshot 2023-05-12 at 12 33 55 PM

**Comment by sshanzel** [comment]
@rebelchris have fixed the description now. Also made the PR a draft to avoid mis-merging until v5 being deployed.

**Review by rebelchris — APPROVED** [review]
Good once we merge :)

**Comment by sshanzel** [comment]
I have pushed a commit to fix the misplaced query parameter for welcoming new members.

https://github.com/dailydotdev/daily-api/pull/1266/commits/25bc45534e5b3e5dc357bbfa7fa746dbfbcf8fa1

**Pull request merged**